pkg/theia/commands/utils_test.go

```diff
@@ -143,3 +143,193 @@ func TestPolicyRecoPreCheck(t *testing.T) {
 		})
 	}
 }
+
+func TestGetClickHouseSecret(t *testing.T) {
```

Comment:
Do we still need `getClickHouseSecret` in this package?

Reply:
No. Do you think I should directly call this func from networkpolicyrecommendation/util.go after #114 merged? Or maybe move it to common pkg, like pkg/util ?

Reply:
I forgot that the ClickHouse status commands still need these common functions. As the implementation of ClickHouse Status commands will be changed in next release, I'm fine to keep the function as it is now.